templates/corporate/billing.html

```diff
@@ -34,7 +34,14 @@ <h1>{{ _("Billing") }}</h1>
                         {% else %}
                         <p>Your current plan is <strong>{{ plan_name }}</strong>.</p>
                         {% endif %}
-                        <p>You are using <strong>{{ licenses_used }} of {{ licenses }} licenses</strong>.</p>
+                        <p>
+                            You are using <strong>{{ licenses_used }} of {{ licenses }} licenses</strong>.
+                            {% if automanage_licenses %}
+                            Your licenses are automatically managed by Zulip.
```

Comment:
I think we should be more clear: "Licenses are automatically managed by Zulip; when you add or deactivate a user, the number of licenses will be immediately updated to reflect your new usage.  <sentence about how handle these adjustments when doing billing at the end of the monthly cycle>"